corehq/apps/consumer_user/templates/consumer_user/change_password.html

```diff
@@ -0,0 +1,56 @@
+{% extends "consumer_user/consumer_user_base.html" %}
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load compress %}
+
+{% block title %}{% trans "Change Password" %}{% endblock title %}
+
+{% block background_content %}
+<div class="bg-container">
+  <div class="bg-full-cover-fixed bg-registration b-lazy b-loaded"
+        style="background-image: url({% static 'hqwebapp/images/molly.jpg' %});"></div>
+  <div class="bg-overlay"></div>
+</div>
+{% endblock %}
+
+{% block stylesheets %}
+{% compress css %}
+<link type="text/less"
+      rel="stylesheet"
+      media="all"
+      href="{% static 'registration/less/registration.less' %}" />
+{% endcompress %}
+{% endblock stylesheets %}
+
+{% block content %}
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-xs-12">
+      <div class="reg-form-container sign-in-container">
+        <div class="form-bubble form-bubble-lg">
+          <h2 class="text-center">{% trans "Change Password" %}</h2>
+          {% if form.errors %}
+          <ul class="user-msg error">
```

Comment:
maybe not worth it, but this looks like it could be extractable to a partial template.